companies/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.1 on 2024-02-13 17:44
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Companies',
+            fields=[
+                ('company_id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('brand', models.CharField(blank=True, max_length=255)),
+                ('is_registered', models.BooleanField(default=False)),
+                ('common_info', models.TextField(blank=True)),
+                ('contact_phone', models.CharField(blank=True, max_length=255)),
+                ('contact_email', models.CharField(blank=True, max_length=255)),
+                ('registration_date', models.DateTimeField(auto_now_add=True)),
+                ('edrpou', models.IntegerField(blank=True)),
```

Comment:
The 'edrpou' field is an IntegerField with blank=True, which could lead to validation issues since integers don't naturally support "blank" as a value. Consider using null=True for optional numeric fields